### ISSUE TITLE
[FEATURE REQUEST] Proper Landscape UI and Foldable Support.

### ISSUE THREAD

**Issue by Mathew20kdroid** [namidaco/namida#202]
As of now the app doesn't have a **proper landscape UI** or the app **don't rotate when phone is in landscape mode**.
I tried to forcibly switch the app to Landscape mode but it made the whole **UI kinda stretched** and the **Now Playing section was completely Broken** , with the top part of the album art going beyond the screen itself.

I would really appreciate if you could make the **app resize to any screen size , may it be a tablet or a foldable.............**

**In Landscape View or on a wider screen it would be nice if the toggles in the now playing section are placed on one side and the album art on the opposite........**
 
![1029372941_Screenshots_2022-09-10-03-07-302 png 0d657d5944c1d7a33d0fbc48f7bf00fe](https://github.com/namidaco/namida/assets/71634955/7ca748b0-0d82-4f4c-bda7-7e46279081b1)

**Comment by MSOB7YY** [comment]
the miniplayer is quite pain to work with so this might take quite a while. also there should be many other related aspects for this to be supported so yeah.

there is also a similar request here to support tablet ui #142

**Comment by MSOB7YY** [comment]
@Mathew20kdroid hey i did whole revamp for allat, its much better now on lanscape/widescreen. (miniplayer still needs some work) but i can try now in v5.1.2 https://github.com/namidaco/namida-snapshots/releases

**Comment by C0C0B01** [comment]
omg there's Landscape finally but it needs a UI fix

**Comment by C0C0B01** [comment]
@Mathew20kdroid pretty sure that is handled by the os

**Comment by MSOB7YY** [comment]
yeah thats handled by the os, the landscape ui is shown once the width is longer than the height by some extent

**Comment by MSOB7YY** [comment]
with v5.1.6 https://github.com/namidaco/namida-snapshots/releases, this should be cool for now. app now properly supports landscape/widescreen, design might be unique/different (as miniplayer will be on the right), but thats what i think is good.
going to close now
